src/main/java/dook/task/Deadline.java

```diff
@@ -0,0 +1,69 @@
+package dook.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+import dook.services.TimeProcessor;
+
+/**
+ * Task with a deadline.
+ */
+public class Deadline extends Task implements TimedTask {
+    protected String by;
```

Comment:
perhaps can make the naming of this field a little more descriptive?

Reply:
Will look into it, thanks for all the feedback!